examples/e2e-tests/example.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test'
+import { fakeTxnResponses } from './FakeAlgodResponses'
+
+test('it works', async ({ page }) => {
+  // Load and set up the page
+  await page.goto('/')
+  await fakeTxnResponses(page)
+  // Whenever a prompt appears, enter the mnemonic
+  page.on('dialog', (dialog) =>
+    dialog.accept(
+      // !! WARN !!
+      // THIS ACCOUNT AND ITS MNEMONIC ARE COMPROMISED.
+      // They are to be used for testing only.
+      // !! WARN !!
+      'sugar bronze century excuse animal jacket what rail biology symbol want craft annual soul increase question army win execute slim girl chief exhaust abstract wink'
+    )
+  )
+
+  // Check mnemonic wallet is activated
+  await expect(page.getByRole('heading', { name: 'Mnemonic' })).toBeVisible()
+
+  // Click the "Connect" button for the Mnemonic wallet
+  await page
+    .getByText('Mnemonic Connect Disconnect')
```

Comment:
The only issue I had testing this locally was that Playwright wasn't finding the right Connect button to click. I was able to get it working by using `.locator(...)` to be a bit more specific isolating the Mnemonic wallet section:
```suggestion
    .locator('.wallet-group', {
      has: page.locator('h4', { hasText: 'Mnemonic' })
    })
```